package-lock.json

```diff
@@ -12841,9 +12841,9 @@
       }
```

Comment:
Strange that we got changes to the `package-lock`. Have you removed it and installed the dependencies again?

Reply:
Yes, there were some packages with vulnerabilities that were updated.

Reply:
Alright!